components/scream/src/physics/p3/tests/p3_main_unit_tests.cpp

```diff
@@ -32,10 +32,16 @@ static void run_phys_p3_main_main_loop()
   // TODO
 }
 
+static void run_phys_p3_main_post_loop()
+{
+  // TODO
```

Comment:
Is this work part of this PR, or a later one?

Reply:
This is a placeholder for a physics (AKA "property") test. They will be added in later PRs.